addons/web/static/src/core/dialog/dialog.xml

```diff
@@ -5,7 +5,7 @@
         <div class="o_dialog" t-att-id="id" t-att-class="{ o_inactive_modal: !data.isActive }">
```

Comment:
Hello @mrsr-odoo 
So that one was not place correctly since the css rule was `.modal.o_inactive_modal`.
I suppose you can remove it then.
Cheers!

Reply:
hello @jeh-odoo 
Removing the class results to various failures in js tests.
As it is used to filter out the inactive dialog in tests.
So keeping it the same
Thanks !!

Reply:
Ok thanks for having a look :slightly_smiling_face: